packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -66,7 +63,7 @@ export class Analyzer implements AnalyzerInterface {
       ...packageInfo,
       modules: rootFileNames.map((fileName) =>
         getModule(
-          this.program.getSourceFile(this.path.normalize(fileName))!,
+          this.path.normalize(fileName) as AbsolutePath,
```

Comment:
Do we have utilities that validate/compute and return the right path types so we don't have to cast?

Reply:
We have an issue for it: https://github.com/lit/lit/issues/2929